eywa/nlu/entity_extractor.py

```diff
@@ -24,6 +28,27 @@ def entities(self):
         return list(self.keys.keys())
 
     def fit(self, X, Y):
+        """
+        Trains the model on given data.
+        # Arguments
+        X: Input utterance(s). It could be:
+            - `str` (or `list` thereof)
+            - `Document` instance (or `list` thereof)
+        Y: Target values. `dict` mapping from from entity name (`str`)
+          to entity value (`str`)(or list thereof).
+          The entity names should be same throughout all `dict` 
+          elements and number of `dict` elements should be
+          same as the number of elements of X.
+        # Example
+        Train an `EntityExtractor` to extract a given entity based on the
```

Comment:
Train an `EntityExtractor` to extract entities "intent" and "place" from utterances.